data/hai missions.txt

```diff
@@ -141,6 +143,7 @@ mission "Ask the Hai about the Unfettered"
 	to offer
 		has "First Contact: Unfettered: offered"
 	on offer
+		log "Spoke to the Hai about their side of the Unfettered's story. If they are to be believed, the ancient Hai lost much of their territory to infighting, and since learned to live in stability and balance."
```

Comment:
`Spoke to the Hai` -> `Spoke to the Hai Elders`?
`their side of the Unfettered's story` -> `what they say about the Unfettered's story`? Or to `their side of the story of the Hai`?
`and since` -> `and have since`